src/Tools/Md2Html/Md2Html.cs

```diff
@@ -81,7 +81,7 @@ private static void ConvertRelativeLocalImagePathsToAbsolute(string mdFilePath,
                     continue;
 
                 var imageName = Path.GetFileName(image.Url);
-                var absoluteImagePath = Path.Combine(VIRTUAL_FOLDER_MAPPING, imageName);
+                var absoluteImagePath = string.Join("/", VIRTUAL_FOLDER_MAPPING, imageName);
```

Comment:
What is the diff here do you mind?

Reply:
@QilongTang with Path.Combine generates something like "http://appassets%502my_image.png", then later I have to apply something like HttpUtility.UrlDecode(absoluteImagePath), so WebView2 can find the image in <img src=, but the problem is that when the src is a base64 content the HttpUtility.UrlDecode() method remove all special characters (that's why nodeautocomplete documentation GIFs are not appearing because has base64 content).
With  string.Join("/" it generates "http://appassets/my_image.png", then there is no need to use UrlDecode so in that case images in base64 and URL are working correctly.

Reply:
@RobertGlobant20 this change has the downside that it's no longer cross platform - can you look at making the same bug fix but in a cross platform way where paths on linux and mac will work?